config/base/rbac/role_binding.yaml

```diff
@@ -10,3 +10,17 @@ subjects:
 - kind: ServiceAccount
   name: controller-manager
   namespace: system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: manager-rolebinding
+  namespace: system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: manager-role
+subjects:
+  - kind: ServiceAccount
+    name: controller-manager
+    namespace: system
```

Comment:
Nit: No new line at the end of this file.

Reply:
Fixed here: https://github.com/operator-framework/operator-controller/commit/bebe28ccf7b4808e046401a36b8aa6af34fdd415